README.md

```diff
@@ -7,23 +7,23 @@ For the list of Proto rules, see the Bazel
 
 ## Getting Started
 
-There is no need to use rules from this repository just yet. If you want to use
-`rules_proto` anyway, add the following to your WORKSPACE file:
+To get started with `rules_proto`, add the following to your `WORKSPACE` file:
 
 ```python
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "rules_proto",
-    urls = ["https://github.com/bazelbuild/rules_proto/archive/TODO"],
-    sha256 = "TODO",
+    sha256 = "<sha256>",
+    strip_prefix = "rules_proto-<version-or-hash>",
+    urls = ["https://github.com/bazelbuild/rules_proto/archive/<version-or-hash>.tar.gz"],
```

Comment:
Maybe put a real sha256 and url? We can update the values periodically (e.g. after each release).

(new users can be confused otherwise)

Reply:
Done. Didn't think this could be an issue until I saw the question on Slack.

(Someone with write permissions to the mirror will need to upload it. See https://github.com/bazelbuild/bazel/pull/8961#discussion_r306378880)